src/main/java/sign_up/use_case/SignUpRequestModel.java

```diff
@@ -0,0 +1,30 @@
+package sign_up.use_case;
+
+public class SignUpRequestModel {
+    // The inputted email for the account
+    public String email;
```

Comment:
These all should be private to avoid any unintentional reassignments, etc. Make these private fields and add getters.